Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "bevy_proto"
-version = "0.2.0"
-edition = "2018"
+version = "0.3.0"
```

Comment:
I prefer bumping versions after PRs are merged. Could we revert this to the current version?